openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -243,6 +243,15 @@ def get_invalid(cls, instance):
                 "{}_render_attributes".format(renderer)) or []
         )
 
+        settings_lights_flag = instance.context.data["project_settings"].get(
+            "maya", {}).get(
+            "create", {}).get(
+            "CreateRender", {}).get(
+            "enable_all_lights", {})
+
+        instance_lights_flag = instance.data.get("RenderSetupIncludeLights")
+        if settings_lights_flag != instance_lights_flag:
+            cls.log.warning('Instance flag for "Render Setup Include Lights" is set to {0} and Settings flag is set to {1}'.format(instance_lights_flag, settings_lights_flag)) # noqa
```

Comment:
Somehow I feel it would be clearer if this would explicitly mention "Project Settings" instead of "Settings"?